<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallingTest</class>
 <widget class="QWidget" name="CallingTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>189</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combobox_name_">
         <item>
          <property name="text">
           <string>Xiaoming</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Phone</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combobox_phone_">
         <item>
          <property name="text">
           <string>13758941236</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14586972375</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13998764523</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13525869542</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="status_lable_">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="status_text_"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushbutton_test_1">
         <property name="text">
          <string>Speaker</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushbutton_test_2">
         <property name="text">
          <string>Reject</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushbutton_test_">
         <property name="text">
          <string>Test</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
